app/_includes/nav-v2.html

```diff
@@ -76,6 +76,11 @@
               </div>
           </ul>
         </li>
+
+        <li role="menuitem" class="main-menu-item">
+          <a href="/api/" class="navbar-item">API Specs</a>
```

Comment:
```suggestion
          <a href="/api/" class="navbar-item">API Documentation</a>
```
Just a small suggestion, I think I've seen more companies use this phrasing

Reply:
@cloudjumpercat , these are just the specs. API docs would be the companion guides, as example. We'll do some cleanup in /konnect/api/ to be more consistent in our terminology.